typespec-tests/src/main/java/com/cadl/flatten/implementation/FlattenClientImpl.java

```diff
@@ -257,7 +257,7 @@ Response<Void> uploadFileSync(@HostParam("endpoint") String endpoint, @PathParam
         @UnexpectedResponseExceptionType(value = ResourceModifiedException.class, code = { 409 })
         @UnexpectedResponseExceptionType(HttpResponseException.class)
         Mono<Response<Void>> uploadTodo(@HostParam("endpoint") String endpoint,
-            @HeaderParam("content-type") String contentType, @HeaderParam("accept") String accept,
+            @HeaderParam("Content-Type") String contentType, @HeaderParam("accept") String accept,
```

Comment:
Is this due to TCGC change? Guessing the lower-case or upper-case do not matter here?

Reply:
More probably dependency lib change like http lib

tsp is
https://github.com/Azure/autorest.java/blob/9c3d8ed0575d107f2e2519033ba67625ecdade85/typespec-tests/tsp/flatten.tsp#L141

If written full it should be `@header("content-type") contentType: "multipart/form-data",`

My guess is that without the explicit header value in `@header`, http lib converts property name `contentType` to header name `Content-Type`.